core/src/com/unciv/ui/newgamescreen/NewGameScreen.kt

```diff
@@ -71,6 +78,13 @@ class NewGameScreen(
         rightSideButton.setText("Start game!".tr())
         rightSideButton.onClick {
             if (gameSetupInfo.gameParameters.isOnlineMultiplayer) {
+                if(!isConnected()){
+                    val invalidPlayerIdPopup = Popup(this)
```

Comment:
Name describes something else

Reply:
I'm not quite sure what you mean here :)